from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(993, 792)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(MainWindow.sizePolicy().hasHeightForWidth())
        MainWindow.setSizePolicy(sizePolicy)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.sliderExperience = QtWidgets.QSlider(self.centralwidget)
        self.sliderExperience.setGeometry(QtCore.QRect(180, 80, 22, 431))
        self.sliderExperience.setMaximum(15)
        self.sliderExperience.setPageStep(1)
        self.sliderExperience.setOrientation(QtCore.Qt.Vertical)
        self.sliderExperience.setObjectName("sliderExperience")
        self.sliderDiplomScore = QtWidgets.QSlider(self.centralwidget)
        self.sliderDiplomScore.setGeometry(QtCore.QRect(410, 80, 22, 431))
        self.sliderDiplomScore.setMinimum(60)
        self.sliderDiplomScore.setMaximum(100)
        self.sliderDiplomScore.setPageStep(1)
        self.sliderDiplomScore.setOrientation(QtCore.Qt.Vertical)
        self.sliderDiplomScore.setObjectName("sliderDiplomScore")
        self.sliderProfessionalTest = QtWidgets.QSlider(self.centralwidget)
        self.sliderProfessionalTest.setGeometry(QtCore.QRect(650, 80, 22, 431))
        self.sliderProfessionalTest.setMaximum(10)
        self.sliderProfessionalTest.setPageStep(1)
        self.sliderProfessionalTest.setOrientation(QtCore.Qt.Vertical)
        self.sliderProfessionalTest.setObjectName("sliderProfessionalTest")
        self.sliderEnglishLevel = QtWidgets.QSlider(self.centralwidget)
        self.sliderEnglishLevel.setGeometry(QtCore.QRect(880, 80, 22, 431))
        self.sliderEnglishLevel.setMaximum(10)
        self.sliderEnglishLevel.setPageStep(1)
        self.sliderEnglishLevel.setOrientation(QtCore.Qt.Vertical)
        self.sliderEnglishLevel.setObjectName("sliderEnglishLevel")
        self.calculateButton = QtWidgets.QPushButton(self.centralwidget)
        self.calculateButton.setGeometry(QtCore.QRect(200, 600, 121, 41))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.calculateButton.setFont(font)
        self.calculateButton.setObjectName("calculateButton")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(140, 40, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label.setFont(font)
        self.label.setAlignment(QtCore.Qt.AlignCenter)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(330, 40, 181, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label_2.setFont(font)
        self.label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(590, 40, 141, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label_3.setFont(font)
        self.label_3.setAlignment(QtCore.Qt.AlignCenter)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(830, 40, 121, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label_4.setFont(font)
        self.label_4.setAlignment(QtCore.Qt.AlignCenter)
        self.label_4.setObjectName("label_4")
        self.labelExperienceValue = QtWidgets.QLabel(self.centralwidget)
        self.labelExperienceValue.setGeometry(QtCore.QRect(150, 530, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelExperienceValue.setFont(font)
        self.labelExperienceValue.setAlignment(QtCore.Qt.AlignCenter)
        self.labelExperienceValue.setObjectName("labelExperienceValue")
        self.labelDiplomScoreValue = QtWidgets.QLabel(self.centralwidget)
        self.labelDiplomScoreValue.setGeometry(QtCore.QRect(380, 530, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelDiplomScoreValue.setFont(font)
        self.labelDiplomScoreValue.setAlignment(QtCore.Qt.AlignCenter)
        self.labelDiplomScoreValue.setObjectName("labelDiplomScoreValue")
        self.labelProfessionalTestValue = QtWidgets.QLabel(self.centralwidget)
        self.labelProfessionalTestValue.setGeometry(QtCore.QRect(620, 530, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelProfessionalTestValue.setFont(font)
        self.labelProfessionalTestValue.setAlignment(QtCore.Qt.AlignCenter)
        self.labelProfessionalTestValue.setObjectName("labelProfessionalTestValue")
        self.labelEnglishLevelValue = QtWidgets.QLabel(self.centralwidget)
        self.labelEnglishLevelValue.setGeometry(QtCore.QRect(850, 530, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelEnglishLevelValue.setFont(font)
        self.labelEnglishLevelValue.setAlignment(QtCore.Qt.AlignCenter)
        self.labelEnglishLevelValue.setObjectName("labelEnglishLevelValue")
        self.labelNumberResult = QtWidgets.QLabel(self.centralwidget)
        self.labelNumberResult.setGeometry(QtCore.QRect(460, 610, 51, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelNumberResult.setFont(font)
        self.labelNumberResult.setAlignment(QtCore.Qt.AlignCenter)
        self.labelNumberResult.setObjectName("labelNumberResult")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(30, 490, 131, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(30, 80, 141, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_6.setFont(font)
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.centralwidget)
        self.label_7.setGeometry(QtCore.QRect(10, 280, 161, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_7.setFont(font)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.centralwidget)
        self.label_8.setGeometry(QtCore.QRect(360, 280, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_8.setFont(font)
        self.label_8.setObjectName("label_8")
        self.label_9 = QtWidgets.QLabel(self.centralwidget)
        self.label_9.setGeometry(QtCore.QRect(310, 490, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_9.setFont(font)
        self.label_9.setObjectName("label_9")
        self.label_10 = QtWidgets.QLabel(self.centralwidget)
        self.label_10.setGeometry(QtCore.QRect(340, 80, 61, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_10.setFont(font)
        self.label_10.setObjectName("label_10")
        self.label_11 = QtWidgets.QLabel(self.centralwidget)
        self.label_11.setGeometry(QtCore.QRect(550, 280, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_11.setFont(font)
        self.label_11.setObjectName("label_11")
        self.label_12 = QtWidgets.QLabel(self.centralwidget)
        self.label_12.setGeometry(QtCore.QRect(560, 490, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_12.setFont(font)
        self.label_12.setObjectName("label_12")
        self.label_13 = QtWidgets.QLabel(self.centralwidget)
        self.label_13.setGeometry(QtCore.QRect(570, 80, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_13.setFont(font)
        self.label_13.setObjectName("label_13")
        self.label_14 = QtWidgets.QLabel(self.centralwidget)
        self.label_14.setGeometry(QtCore.QRect(790, 280, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_14.setFont(font)
        self.label_14.setObjectName("label_14")
        self.label_15 = QtWidgets.QLabel(self.centralwidget)
        self.label_15.setGeometry(QtCore.QRect(830, 490, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_15.setFont(font)
        self.label_15.setObjectName("label_15")
        self.label_16 = QtWidgets.QLabel(self.centralwidget)
        self.label_16.setGeometry(QtCore.QRect(810, 80, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_16.setFont(font)
        self.label_16.setObjectName("label_16")
        self.sliderResult = QtWidgets.QSlider(self.centralwidget)
        self.sliderResult.setGeometry(QtCore.QRect(40, 690, 881, 22))
        self.sliderResult.setMaximum(5)
        self.sliderResult.setSingleStep(1)
        self.sliderResult.setPageStep(1)
        self.sliderResult.setOrientation(QtCore.Qt.Horizontal)
        self.sliderResult.setObjectName("sliderResult")
        self.label_17 = QtWidgets.QLabel(self.centralwidget)
        self.label_17.setGeometry(QtCore.QRect(20, 730, 81, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_17.setFont(font)
        self.label_17.setObjectName("label_17")
        self.label_18 = QtWidgets.QLabel(self.centralwidget)
        self.label_18.setGeometry(QtCore.QRect(860, 730, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_18.setFont(font)
        self.label_18.setObjectName("label_18")
        self.label_20 = QtWidgets.QLabel(self.centralwidget)
        self.label_20.setGeometry(QtCore.QRect(340, 730, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_20.setFont(font)
        self.label_20.setObjectName("label_20")
        self.label_21 = QtWidgets.QLabel(self.centralwidget)
        self.label_21.setGeometry(QtCore.QRect(510, 730, 101, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_21.setFont(font)
        self.label_21.setObjectName("label_21")
        self.label_19 = QtWidgets.QLabel(self.centralwidget)
        self.label_19.setGeometry(QtCore.QRect(390, 610, 71, 20))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label_19.setFont(font)
        self.label_19.setAlignment(QtCore.Qt.AlignCenter)
        self.label_19.setObjectName("label_19")
        self.labelTermResult = QtWidgets.QLabel(self.centralwidget)
        self.labelTermResult.setGeometry(QtCore.QRect(670, 610, 151, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.labelTermResult.setFont(font)
        self.labelTermResult.setAlignment(QtCore.Qt.AlignCenter)
        self.labelTermResult.setObjectName("labelTermResult")
        self.label_22 = QtWidgets.QLabel(self.centralwidget)
        self.label_22.setGeometry(QtCore.QRect(560, 610, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.label_22.setFont(font)
        self.label_22.setAlignment(QtCore.Qt.AlignCenter)
        self.label_22.setObjectName("label_22")
        self.label_23 = QtWidgets.QLabel(self.centralwidget)
        self.label_23.setGeometry(QtCore.QRect(160, 730, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_23.setFont(font)
        self.label_23.setObjectName("label_23")
        self.label_24 = QtWidgets.QLabel(self.centralwidget)
        self.label_24.setGeometry(QtCore.QRect(680, 730, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.label_24.setFont(font)
        self.label_24.setObjectName("label_24")
        MainWindow.setCentralWidget(self.centralwidget)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Suitability assessment"))
        self.calculateButton.setText(_translate("MainWindow", "Calculate"))
        self.label.setText(_translate("MainWindow", "Experience"))
        self.label_2.setText(_translate("MainWindow", "Average diplom score"))
        self.label_3.setText(_translate("MainWindow", "Professional test"))
        self.label_4.setText(_translate("MainWindow", "English level"))
        self.labelExperienceValue.setText(_translate("MainWindow", "0"))
        self.labelDiplomScoreValue.setText(_translate("MainWindow", "60"))
        self.labelProfessionalTestValue.setText(_translate("MainWindow", "0"))
        self.labelEnglishLevelValue.setText(_translate("MainWindow", "0"))
        self.labelNumberResult.setText(_translate("MainWindow", "---"))
        self.label_5.setText(_translate("MainWindow", "Without experience"))
        self.label_6.setText(_translate("MainWindow", "Extensive experience"))
        self.label_7.setText(_translate("MainWindow", "Considerable experience"))
        self.label_8.setText(_translate("MainWindow", "Good"))
        self.label_9.setText(_translate("MainWindow", "Satisfactory"))
        self.label_10.setText(_translate("MainWindow", "Excellent"))
        self.label_11.setText(_translate("MainWindow", "Medium score"))
        self.label_12.setText(_translate("MainWindow", "Low score"))
        self.label_13.setText(_translate("MainWindow", "High score"))
        self.label_14.setText(_translate("MainWindow", "Intermediate"))
        self.label_15.setText(_translate("MainWindow", "Basic"))
        self.label_16.setText(_translate("MainWindow", "Advanced"))
        self.label_17.setText(_translate("MainWindow", "Not suitable"))
        self.label_18.setText(_translate("MainWindow", "Excellent suitable"))
        self.label_20.setText(_translate("MainWindow", "Maybe suitable"))
        self.label_21.setText(_translate("MainWindow", "Partially suitable"))
        self.label_19.setText(_translate("MainWindow", "Result:"))
        self.labelTermResult.setText(_translate("MainWindow", "---"))
        self.label_22.setText(_translate("MainWindow", "Inference:"))
        self.label_23.setText(_translate("MainWindow", "Unlikely suitable"))
        self.label_24.setText(_translate("MainWindow", "Probably suitable"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    ui = Ui_MainWindow()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec_())
